File: src/main/java/frc/team4069/robot/commands/vacuum/StartVacuumCommand.java
package frc.team4069.robot.commands.vacuum;

import frc.team4069.robot.commands.CommandBase;

public class StartVacuumCommand extends CommandBase {
    public StartVacuumCommand() {
        requires(vacuum);
    }

    @Override
    protected void initialize() {
        vacuum.start();
    }

    @Override
    protected boolean isFinished() {
        return true;
    }
}

File: src/main/java/frc/team4069/robot/commands/vacuum/StartVacuumCommand.kt
package frc.team4069.robot.commands.vacuum

import edu.wpi.first.wpilibj.command.InstantCommand
import frc.team4069.robot.subsystems.VacuumSubsystem

class StartVacuumCommand : InstantCommand() {
    init {
        requires(VacuumSubsystem)
    }

    override fun initialize() = VacuumSubsystem.start()
}
